client/components/domains/transfer-domain-step/index.jsx

```diff
@@ -179,26 +179,10 @@ class TransferDomainStep extends React.Component {
 						<div className="transfer-domain-step__domain-heading">
 							{ translate( 'Manage your domain and site together on WordPress.com.' ) }
 						</div>
-						<div className="transfer-domain-step__domain-text">
-							{ translate(
-								'Transfer your domain away from your current provider to WordPress.com so you can update settings, ' +
-									"renew your domain, and more \u2013 right in your dashboard. We'll renew it for another year " +
-									'when the transfer is successful. {{a}}Learn more{{/a}}',
-								{
-									components: {
-										a: (
-											<a
-												href={ INCOMING_DOMAIN_TRANSFER }
-												rel="noopener noreferrer"
-												target="_blank"
-											/>
-										),
-									},
-								}
-							) }
-						</div>
 					</div>
 
+					<div className="domain-product-price">{ domainProductPrice }</div>
```

Comment:
I think this causing tests to fail. We should rename class: `transfer-domain-step__ prefix`

Reply:
![screen shot 2018-07-26 at 10 00 10 am](https://user-images.githubusercontent.com/4924246/43276774-baad5026-90ba-11e8-9208-44c853ed5733.png)

Reply:
I think it's just looking for `transfer-domain-step__` as a prefix, not necessarily needing `prefix` in the class name. Something like `transfer-domain-step__price` would make sense I think.

Reply:
Aha, good point. I was trying to reuse the same class name between the two sections for consistency, but one uses a Component and the other does not. Will change that!

Reply:
Yeah, sorry, I meant to say the class needs to be prefixed with  transfer-domain-step__` :)